environment/test_helpers.go

```diff
@@ -0,0 +1,314 @@
+package environment
```

Comment:
does this break down if we put it in environment_test?

Reply:
quite a lot -- as some functions i am using are not public. Happy to move them too https://github.com/dagger/container-use/pull/65#discussion_r2148877778